pom.xml

```diff
@@ -25,7 +25,7 @@
 
   <properties>
     <java.level>8</java.level>
-    <jackson.version>2.8.9</jackson.version>
+    <jackson.version>2.8.10</jackson.version>
     <jenkins.version>2.73.3</jenkins.version> <!-- Should be kept in sync with Dockerfile FROM statement-->
```

Comment:
this and the ATH version in the Jenkinsfile *may* need to be bumped to latest LTS if some of these new deps require it